### PR TITLE
Onboarding logic

### LINKED ISSUE #382: Integrate onboarding screens into app

### PR DESCRIPTION
**Pull request by tanx**
Closes #382

### CODE REVIEW

File: src/action/wallet.js [hunk 1]
```diff
@@ -1,13 +1,66 @@
...
+  // Wallet actions
+  //
+
+  async init() {
+    try {
+      await this.generateSeed();
+      this._nav.goLoader();
+      await nap(NOTIFICATION_DELAY);
+      this._nav.goSeed();
+    } catch (err) {
```
Comment (valentinewallace) [inline src/action/wallet.js]: I ran into an error here: "wallet already exists" -- had to delete my existing `.config/lightning-app` to get to the onboarding.

Should we be logging this error to the console and/or putting a note in the README for people who have previously downloaded the app?

File: src/computed/seed.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ const getSeedIndexes = (min, max, numWords) => {
     let foundValid = false;
     while (!foundValid) {
       let candidate = Math.floor(Math.random() * (max - min + 1)) + min;
-      if (indices.indexOf(candidate) == -1) {
+      if (indices.indexOf(candidate) === -1) {
```
Comment (valentinewallace) [inline src/computed/seed.js]: ah yeah forgot about the === thing, noted for the future!

File: src/view/new-address.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const NewAddressView = ({ store, invoice }) => (
+const NewAddressView = ({ store, nav, invoice }) => (
```
Comment (valentinewallace) [inline src/view/new-address.js]: For me, the address itself didn't show up for this view:
no_address_showing_err

File: src/view/seed-verify.js [hunk 4]
```diff
@@ -46,16 +46,23 @@ const SeedVerifyView = ({ store }) => (
...
+          <SeedEntry
+            seedIndex={seedIndex}
+            value={store.wallet.seedVerify[i]}
+            onChangeText={word => wallet.setSeedVerify({ word, index: i })}
+            key={i}
+          />
         ))}
       </Card>
-      <GlasButton onPress={() => {}}>Next</GlasButton>
+      <GlasButton onPress={() => wallet.checkSeed()}>Next</GlasButton>
```
Comment (valentinewallace) [inline src/view/seed-verify.js]: I was thinking of adding an issue for the `Return` key being equivalent to pressing `Next` (applies to multiple screens)

**Comment by tanx** [comment]
> being able to fully demo the onboarding process brings a tear to my eye :')

LOL. Yeah great work 👍

File: src/action/wallet.js (hunk 1 above)
Comment (tanx) [reply src/action/wallet.js]: I think it should be good. Not too many testers yet.

File: src/view/new-address.js (hunk 3 above)
Comment (tanx) [reply src/view/new-address.js]: Created an issue: https://github.com/lightninglabs/lightning-app/issues/394

**Pull request merged**